### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by anthony1x6000** [anthony1x6000/ROBLOX2016stylus#59] [Announcements]
To keep this repository active, I would like some maintainers to keep things updated with Roblox's ongoing site changes. 
After exams I'll work on getting everything I can fixed. However, when my second semester starts, this repository will likely remain untouched by myself for quite some time. 
As always, any support, even in the form of pull requests and issues are always appreciated. 

Maintainers would hopefully fix issues and approve pull requests, along with ideally being active on Roblox.

**Comment by parker02311** [comment]
@anthony1x6000 just saw this, I would be willing to help out, but I am unaware of any current issues.

**Reply by anthony1x6000** [reply]
Hey, I appreciate the reply. 

There aren't any urgent issues currently, I'm just looking for people who are active on Roblox and can keep this style maintained by patching up new features on the Roblox website, ensuring they look faithful to 2016.
For example, the "Today's Picks" thing on the home page didn't conform to the rest of the style. I'd like for maintainers to, ideally, first make it faithful to 2016, or potentially just remove it entirely. 

Let me know if you'd want to go through with being a maintainer, you don't have to be a professional at CSS or anything, I'm far from it myself, just knowing the basics and stuff. You'd just need the patience to go through the messy userstyle that it currently is right now to implement larger scale fixes - I apologize.

**Reply by parker02311** [reply]
I wouldn't mind maintaining while your unable to. As almost all of my interests currently rely on the Roblox platform I would be able to spot most bugs, and I would keep up with issues/prs.

Just for the month of December I am going to be unable to do any major things, just minor bug fixes, and possible working on the "Today's Picks". 
Is there a better place we can get in contact? 

Edit: Looks like you have already solved "Today's Picks"

**Reply by anthony1x6000** [reply]
Thank you, I'll add you as a collaborator. 
Discord is a good place for contact, let me know if you're able to talk through there.

**Reply by parker02311** [reply]
Yes, Discord is perfect. My username is just `parker02311`, if you send a friend request, let me know what your username is so I can accept it.

**Reply by anthony1x6000** [reply]
Hey, just sent a request. My bio is "draining," I also have my website under "Connections."
